front/package.json

```diff
@@ -34,7 +36,9 @@
     "dependency-cruiser": "depcruise app  --validate",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
-    "chromatic": "chromatic --exit-once-uploaded"
+    "chromatic": "chromatic --exit-once-uploaded",
+    "generate-component-library-types": "tsc app/component-library/index.tsx --jsx react --esModuleInterop true --skipLibCheck --emitDeclarationOnly --declaration --declarationDir app/component-library/dist",
```

Comment:
I kept these scripts in the main package json because I don't want to have dependencies in the other one.